test/game_utility.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Player } from '../src/Player';
+import { Loiterer } from '../src/Loiterer';
+import { Spymaster } from '../src/Spymaster';
+import { Operative } from '../src/Operative';
+import { GameUtility as gu } from '../src/GameUtility';
+import { Color, Team, Turn } from '../src/constants/Constants';
+
+import 'mocha';
+import { expect } from 'chai';
+import WebSocket = require('ws')
+import { mock, instance, when } from 'ts-mockito';
+import { SPlayerTeams } from '../src/Teams';
+
+describe("Filename: game_utility.test.ts:\n\nGame Utility", () => {
+	let mock_ws: WebSocket;
+	let mock_ws_instance: WebSocket;
+	let loiterers: Loiterer[];
+	let players: Player[];
+
+	before(() => {
+		mock_ws = mock(WebSocket);
+		mock_ws_instance = instance(mock_ws);
+
+		let red_l_one = new Loiterer("red", "1", Team.red, mock_ws_instance);
+		let red_l_two = new Loiterer("red", "2", Team.red, mock_ws_instance);
+		let blue_l_one = new Loiterer("blue", "1", Team.blue, mock_ws_instance);
+		let blue_l_two = new Loiterer("blue", "2", Team.blue, mock_ws_instance);
+		let blue_l_three = new Loiterer("blue", "3", Team.blue, mock_ws_instance);
+		loiterers = [red_l_one, red_l_two, blue_l_one, blue_l_two, blue_l_three];
+
+		let red_p_one = new Spymaster("red", "1", Team.red, mock_ws_instance);
+		let red_p_two = new Operative("red", "2", Team.red, mock_ws_instance);
+		let red_p_three = new Operative("red", "3", Team.red, mock_ws_instance);
+		let blue_p_one = new Spymaster("blue", "1", Team.blue, mock_ws_instance);
+		let blue_p_two = new Operative("blue", "2", Team.blue, mock_ws_instance);
+		players = [red_p_one, red_p_two, red_p_three, blue_p_one, blue_p_two];
+	});
+
+  it("should correctly split loiterers on team", () => {
```

Comment:
Make sure to consistently use one type of indenting. Tab based indentation is used everywhere else in this file, but this line (and a few others) uses two spaces.

Reply:
Will resolve in one monster commit later.